extension/src/experiments/webview/messages.ts

```diff
@@ -376,9 +376,10 @@ export class WebviewMessages {
   }
 
   private async setMaxTableHeadDepth() {
+    const currentValue = getConfigValue(ConfigKey.EXP_TABLE_HEAD_MAX_HEIGHT)
     const newValue = await getPositiveIntegerInput(
       Title.SET_EXPERIMENTS_HEADER_HEIGHT,
-      { prompt: 'Use 0 for infinite height.', value: '0' },
+      { prompt: 'Use 0 for infinite height.', value: currentValue },
```

Comment:
At the start of the "PR demo" there is no config option set but "5" is shown for the value. How did that happen? Should this be `currentValue ?? '0'` or similar?

Reply:
Oh, the default is set in `package.json`

```
        "dvc.experimentsTableHeadMaxHeight": {
          "description": "Maximum height of Experiment table head rows. Use 0 for infinite height.",
          "type": "number",
          "default": 5
        }
```

Reply:
Actually, I kind of assumed `5` must have been set as the default value somewhere as it's the same in Studio. But, I didn't know where, so thanks for sharing 😄.